composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
     	"php": ">=5.5.0",
         "guzzlehttp/guzzle": "^6.1",
-        "psr/log": "1.0.0",
+        "psr/log": "~1.0",
```

Comment:
This makes absolutely no difference in that case. But I'll push a new version for you ;-)